File: Tamago-FrontEnd/src/app/Models/AnimalModel.ts
import { EntitieBase } from './EntitieBase';

export class AnimalModel extends EntitieBase {
  public id: string;
  public userId: string;
  public name: string;
  public food: number;
  public energy: number;
  public happiness: number;
  public animalType: number;
  public level: number;

  constructor(id, Uid, n, f, e, h, type, level) {

    super();

    this.id = id;
    this.userId = Uid;
    this.name = n;
    this.food = f;
    this.energy = e;
    this.happiness = h;
    this.animalType = type;
    this.level = level;
  }

}

File: Microservices Demo/Shared/ApiModels/AnimalModel.cs
using System;
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;
using System.Text;

namespace Shared.Shared.ApiModels
{
    [ExcludeFromCodeCoverage]
    public class AnimalModel
	{
        public Guid Id { get; set; }
        public Guid UserId { get; set; }
        public string Name { get; set; }
        public float Food { get; set; }
        public float Energy { get; set; }
        public float Happiness { get; set; }
        public int AnimalType { get; set; }
    }
}
